XenModel/Actions/Updates/DownloadAndUnzipXenServerPatchAction.cs

```diff
@@ -64,13 +64,13 @@ public DownloadAndUnzipXenServerPatchAction(string patchName, Uri uri, string ou
             : this(patchName, uri, outputFileName, InvisibleMessages.XEN_UPDATE)
         { }
 
-        public DownloadAndUnzipXenServerPatchAction(string patchName, Uri uri, string outputFileName, string updateFileExtension)
+        public DownloadAndUnzipXenServerPatchAction(string patchName, Uri uri, string outputFileName, params string[] updateFileExtensions)
             : base(null, string.Format(Messages.DOWNLOAD_AND_EXTRACT_ACTION_TITLE, patchName), string.Empty, false)

```

Comment:
This action is always used with suppressHistory=false. I think we should expose this parameter and call it with suppressHistory=true from the Select Update page